COMMITTERS.adoc

```diff
@@ -0,0 +1,164 @@
+  you can also (carefully) do it for him as soon as you agreed on it.
+* make sure the commit comments are meaningful
+* at the beginning, you can start by giving your approval and not merging
+  and see how things evolve.
+  This should give you some hints of important things to check in the future.
+* only merge pull requests that target master.
+  Dealing with old branches is a separate process.
+* if you are sure a PR should go in and is just waiting for CI,
+  just use the `triage/waiting-for-ci` label.
+  It is a good hint for other committers.
+
+Obviously, each situation is different so use your own judgement,
+and if in doubt, just ask for advice, the other committers are here to help.
+
+== Pull Request Title
+
+The title should be in English, should not contain an issue number.
+It should also not contain ellipsis.
+
+If your commit message was too long and GitHub automatically cut
+the title, that's helpful if you can take the time to move the cut
+part where it belongs to have a full title.
+
+Titles are included in the Release notes so they are important.
+
+A good title would look like: `Fix XXX in Quartz extension`.
+
+A bad title would look like: `fix(#444)`.
+
+== Issues Fixed
+
+When a PR fixes some issues, it's good practice to add it in the description.
+
+One issue per line with something like:
+
+[source,asciidoc]
+----
+* Fix #444
+* Fix #555
+----
+
+Note that GitHub won't detect issues properly if you do something like
+`Fix #444 #555`.
+
+== Affecting Labels and Milestones
+
+Affecting labels and milestones is very important in our process.
+
+Before each release, we check that we don't have any orphan i.e.
+closed issues that don't have either a milestone affected or
+some of the "excluding" labels.
+
+Thus:
+
+* when your merge a pull request, please affect it with the current milestone marked with "master".
+* if the pull request has issues associated (i.e. if the pull request fixes some issues),
+  also affect the milestone to the issues.
+* if you close a pull request because we won't merge it, please add the appropriate `triage/` label:
+  `triage/invalid`, `triage/out-of-date`, `triage/wontfix` are usually in order.
+* if you close an issue because it has been fixed, either add the milestone
+  if you can find it easily or use the `triage/out-of-date` if you can't.
+* if you close an issue for any other reason, one of the aforementioned `triage/` labels
+  is probably adequate.
+
+== Backporting Process
+
+When we release a new version of Quarkus, we usually do a bugfix
+release a couple of weeks after.
+
+Everytime, we do a release, we create a release branch to host
+the commits for these bugfix releases.
+
+All the pull requests are merged in master so for the new feature 
+release of Quarkus.
+They won't be integrated in the previous version branch.
+
+One of your pull request might qualify for being backported to this
+bugfix branch.
+
+Good examples of that might be:
+
+* bugfixes
```

Comment:
```suggestion
* bug fixes
```

Reply:
not sure about that one, I think it's commonly accepted.